pkg/pgmodel/model/series.go

```diff
@@ -98,6 +99,13 @@ func (l *Series) IsSeriesIDSet() bool {
 	return l.isSeriesIDSetNoLock()
 }
 
+//FinalSizeBytes returns the size in bytes /after/ the seriesID is set
+func (l *Series) FinalSizeBytes() uint64 {
+	//size is the base size of the struct + the str and metricName strings
+	//names and values are not counted since they will be nilled out
+	return uint64(unsafe.Sizeof(*l)) + uint64(len(l.str)+len(l.metricName)) // #nosec
```

Comment:
I think we should avoid unsafe package in our builds. I may be wrong but I heard that they have security vulnerabilities.

https://dev.to/jlauinger/exploitation-exercise-with-unsafe-pointer-in-go-information-leak-part-1-1kga

Reply:
This is more easy to understand my point

https://hackernoon.com/golang-unsafe-type-conversions-and-memory-access-odz3yrl